src/python/WMCore/MicroService/Unified/MSCore.py

```diff
@@ -68,53 +65,12 @@ def change(self, reqName, reqStatus, prefix='###'):
         except Exception as err:
             self.logger.exception("Failed to change request status. Error: %s", str(err))
 
-    def updateTransferInfo(self, requestStatuses):
-        "Update transfer ids in backend"
-        tstamp = time.time()
-        # Alan's suggestion to use bulk query
-        # get all docs from CouchDB
-        docs = self.getTransferInfo('ALL_DOCS')
-        for wname, statusRecord in requestStatuses:
-            # obtain records from CouchDB
-            # doc = self.getTransferInfo(wname)
-            # find record in our list of docs
-            # VK: I don't know if it will be faster then fetching the record
-            #     in CouchDB, we need to loop every time to find proper record
-            doc = {}
-            for doc in docs:
-                if wname in doc:
-                    break
-            records = doc.get('transfers', [])
-            records += statusRecord
-            doc['workflowName'] = wname
-            doc['timestamp'] = tstamp
-            doc['transfers'] = records
-            self.reqmgrAux.postTransferInfo(wname, doc)
-
-    def getTransferInfo(self, wname):
-        """
-        Get transfer document from backend. The document has the following form:
-
-        .. doctest::
-
-            {"workflowName": "bla",
-             "timestamp": 123,
-             "transfers": [rec1, rec2, ... ]
-            }
-            where each record has the following format:
-            {"dataset":"/a/b/c", "dataType": "primary", "transferIDs": [1,2], "completion": 0}
-
-
-        :param wname: workflow name
-        :return: a tranfer ids document
-        """
-        return self.reqmgrAux.getTransferInfo(wname)
 
     def getTransferIds(self, dataset):
```

Comment:
This one doesn't belong here as well and we should move it to MSMonitor.
In addition to that, it's not getting any transfer ID, but it's actually getting the subscription status for a dataset and we should change things accordingly.
Last but not least, given that the phedex status will be done via GET call, we should try to use `multi_getdata` and get things in parallel.